modules/UI/videolayout/SmallVideo.js

```diff
@@ -273,7 +274,12 @@ SmallVideo.prototype.setVideoMutedView = function(isMuted) {
 
     var videoMutedSpan = this.getVideoMutedIndicator();
 
-    videoMutedSpan[isMuted ? 'show' : 'hide']();
+    if (isMuted) {
+        // hide and close tooltip
+        videoMutedSpan.hide().trigger('mouseleave');
```

Comment:
If there's no way around this we should at least add a general show and hide functions in UIUtil that make sure the tooltip is taken care of (call mouseleave or directly remove the tooltip for the element).